tests/test_utils.py

```diff
@@ -227,24 +227,24 @@ def test_stdsim_line_buffering(base_app):
 def pr_none():
     import subprocess
 
+    # Start a long running process so we have time to run tests on it before it finishes
     # Put the new process into a separate group so signals sent to it won't interfere with this process
     if sys.platform.startswith('win'):
-        command = 'dir'
+        command = 'timeout -t -1 /nobreak'
```

Comment:
Fantastic work fixing some unreliable tests which have been giving us grief for a long time!